src/textures/CubeTexture.js

```diff
@@ -39,20 +46,20 @@ Object.defineProperty( CubeTexture.prototype, 'images', {
 
 CubeTexture.prototype.fromEquirectangular = function( renderer, source, size ) {
 
-	var scene = new THREE.Scene();
+	var scene = new Scene();
 
 	var gl = renderer.getContext();
 	var maxSize = gl.getParameter( gl.MAX_CUBE_MAP_TEXTURE_SIZE )
 
-	var camera = new THREE.CubeCamera( 1, 100000, Math.min( size, maxSize ) );
+	var camera = new CubeCamera( 1, 100000, Math.min( size, maxSize ) );
 
-	var material = new THREE.MeshBasicMaterial( {
+	var material = new MeshBasicMaterial( {
 		map: source,
-		side: THREE.BackSide
+		side: BackSide
 	} );
 
-	var mesh = new THREE.Mesh(
-		new THREE.IcosahedronGeometry( 100, 4 ),
+	var mesh = new Mesh(
+		new IcosahedronGeometry( 100, 4 ),
```

Comment:
Good that added detail param. I'm a bit concerned about poles though vs. using a sphere even though bad vertex utilization, obviously very dependent on image: https://jsfiddle.net/fgbx0pb3/ sphere, ico:3, ico:4.

 Could be improved later though. Could implement ico subdivision with greater granularity (http://donhavey.com/blog/tutorials/tutorial-3-the-icosahedron-sphere/), or different sphere geometry generation.